Source/WebKit/Shared/API/c/WKContextMenuItem.cpp

```diff
@@ -194,7 +194,7 @@ STATIC_ASSERT_EQUALS(16, kWKContextMenuItemTagNoGuessesFound, ContextMenuItemTag
 STATIC_ASSERT_EQUALS(17, kWKContextMenuItemTagIgnoreSpelling, ContextMenuItemTagIgnoreSpelling);
 STATIC_ASSERT_EQUALS(18, kWKContextMenuItemTagLearnSpelling, ContextMenuItemTagLearnSpelling);
 STATIC_ASSERT_EQUALS(19, kWKContextMenuItemTagOther, ContextMenuItemTagOther);
-STATIC_ASSERT_EQUALS(20, kWKContextMenuItemTagSearchInSpotlight, ContextMenuItemTagSearchInSpotlight);
+static_assert(20 == kWKContextMenuItemTagSearchInSpotlight);
```

Comment:
Why change the format on this one? Just because we're only doing one compare?

Reply:
right, there's no third value